docs/investigation/local-hub-v2/ts-grpc/grpc-gen/README.md

```diff
@@ -0,0 +1,85 @@
+# grpc-proto-loader example
+
+This example shows how to use different libraries to generate a gRPC client. Used libraries:
+
+- [`@grpc/proto-loader`](https://www.npmjs.com/package/@grpc/proto-loader)
+- [`ts-proto`](https://github.com/stephenh/ts-proto)
+
+## App layout
+
+- [client-official.ts](client-official.ts) - Showcase usage of gRPC client generated via [`@grpc/proto-loader`](https://www.npmjs.com/package/@grpc/proto-loader).
+- [client-official-await.ts](client-official-await.ts) - Showcase async/await usage of gRPC client generated via [`@grpc/proto-loader`](https://www.npmjs.com/package/@grpc/proto-loader). Promises are implemented manually.
+- [client-ts-plugin.ts](client-ts-plugin.ts) - Showcase async/await usage of gRPC client generated via [`ts-proto`](https://github.com/stephenh/ts-proto). Promises are provided via [`nice-grpc`](https://www.npmjs.com/package/nice-grpc).
+
+## Generating the clients
+
+Install dependencies:
+
+```bash
+npm install
+```
+
+### [`@grpc/proto-loader`](https://www.npmjs.com/package/@grpc/proto-loader)
+
+To generate the TypeScript files into [`./proto/official`](./proto/ts-plugin), run:
+
+```bash
+$(npm bin)/proto-loader-gen-types --longs=String --enums=String --defaults --oneofs --outDir=proto/official ../../../../../hub-js/proto/storage_backend.proto
```

Comment:
1. It is missing grpcLib flag
2. You can use `npx` to run local and remote packages:
```suggestion
npx proto-loader-gen-types --longs=String --enums=String --defaults --oneofs --outDir=proto/official 
--grpcLib=@grpc/grpc-js ../../../../../hub-js/proto/storage_backend.proto
```